scripts/compile_version_upload_s3.sh

```diff
@@ -138,22 +116,21 @@ docker cp $CONTAINER_ID:/home/docs/$OS-$TOOL-$VERSION.tar.gz .
 # Kill the container
 docker container kill $CONTAINER_ID
 
-# Upload the .tar.gz to S3
-AWS_ENDPOINT_URL="${AWS_ENDPOINT_URL:-http://localhost:9000}"
-AWS_BUILD_TOOLS_BUCKET="${AWS_BUILD_TOOLS_BUCKET:-build-tools}"
-AWS_ACCESS_KEY_ID="${AWS_ACCESS_KEY_ID:-admin}"
-AWS_SECRET_ACCESS_KEY="${AWS_SECRET_ACCESS_KEY:-password}"
-
-if [[ -z $AWS_REGION ]]
+if [[ -z $CIRCLECI ]]
```

Comment:
I noticed this on the shell scripting on the ops side of this change as well, and a few spots above, but just to note: when using variables in sh/bash, it's best to quote them:

https://tldp.org/LDP/abs/html/quotingvar.html

We don't expect are any problematic characters in these variables, so this isn't an issue.

I've definitely been bit by unquoted variables in the past, usually around string comparisons.

```suggestion
if [[ -z "$CIRCLECI" ]]
```